### PR TITLE
feat: [IOCOM-922] Remove medical prescription

### PR DESCRIPTION
**Pull request by adelloste**
## Short description
This PR removes Medical prescription from the project

## List of changes proposed in this pull request
- removed Medical prescription related files
- removed translation keys

## How to test
Tests should be solved correctly

### CODE REVIEW

File: ts/features/messages/components/MessageDetail/MessageTitle.tsx [hunk 1]
```diff
@@ -1,24 +1,8 @@
...
-export const MessageTitle = (props: Props): React.ReactElement =>
-  props.isPrescription ? (
-    <>
-      <H1>{I18n.t("messages.medical.prescription")}</H1>
-      <Body>{I18n.t("messages.medical.memo")}</Body>
-    </>
-  ) : (
-    <H1>{props.title}</H1>
-  );
+export const MessageTitle = ({ title }: MessageTitleProps) => <H1>{title}</H1>;
```
Comment (Vangaorth) [inline ts/features/messages/components/MessageDetail/MessageTitle.tsx:8]: Should we delete this component and move the `<H1>{title}</H1>` into the component that is referencing it?

File: ts/features/messages/components/MessageDetail/HeaderDueDateBar.tsx [hunk 2]
```diff
@@ -1,45 +1,28 @@
...
-export const HeaderDueDateBar = (props: Props): React.ReactElement => {
-  const paymentExpirationInfo = getPaymentExpirationInfo(props.messageDetails);
+export const HeaderDueDateBar = ({
+  hasPaidBadge,
+  messageDetails
+}: HeaderDueDateBarProps) => {
+  const paymentExpirationInfo = getPaymentExpirationInfo(messageDetails);
 
   return (
     <>
```
Comment (Vangaorth) [inline ts/features/messages/components/MessageDetail/HeaderDueDateBar.tsx:18]: Should we remove this fragment?

**Review by Vangaorth — CHANGES_REQUESTED** [review]

File: ts/features/messages/components/MessageDetail/MessageTitle.tsx (hunk 1 above)
Comment (adelloste) [reply ts/features/messages/components/MessageDetail/MessageTitle.tsx]: Applied https://github.com/pagopa/io-app/pull/5407/commits/f421d75cbfcbde91d2ce52b6952e61d9e93e8a92

File: ts/features/messages/components/MessageDetail/HeaderDueDateBar.tsx (hunk 2 above)
Comment (adelloste) [reply ts/features/messages/components/MessageDetail/HeaderDueDateBar.tsx]: Applied https://github.com/pagopa/io-app/pull/5407/commits/e46c41718336a23b016908d22e39f18cd64a177e

**Review by Vangaorth — APPROVED** [review]
LGTM

**Pull request merged**